beaker-tests/Sanity/copr-cli-basic-operations/runtest-modules.sh

```diff
@@ -85,7 +85,7 @@ function test_successful_packages()
 rlJournalStart
     rlPhaseStartSetup
         setup_checks
-        yum -y install dnf dnf-plugins-core
+        yum -y install dnf dnf-3 dnf-plugins-core
```

Comment:
I never seen `dnf-3` :D interesting.

Having `dnf install dnf4 dnf3` would be slightly more obvios.

Reply:
on fedora 41:

```
[root@10a47da1ac97 /]# dnf install dnf3
Updating and loading repositories:
Repositories loaded.
Failed to resolve the transaction:
No match for argument: dnf3
You can try to add to command line:
  --skip-unavailable to skip unavailable packages
[root@10a47da1ac97 /]#
```

Reply:
s/dnf-3/dnf4